README.md

```diff
@@ -119,7 +119,7 @@ The Actions runners need a Docker image to launch the pods that will run them. T
     2. Update the `kustomizations.yaml` file to point to the production cluster in the paths of the Kustomizations
     3. You need to replace/fill in values for two deployments:
         1. `aws-system/aws-auth.yaml`
-            1. Replace the values for the two role ARNs with the ones from the pulumi output. If you need to get the output again you can run `pulumi stack output` and it will print the values. The first value is for the Linux ARN and the second value is for the Windows ARN.
+            1. If you are creating a single node type you can delete the `aws-system/aws-auth.yaml` file and proceed otherwise replace the values for the two role ARNs with the ones from the pulumi output. If you need to get the output again you can run `pulumi stack output` and it will print the values. The first value is for the Linux ARN and the second value is for the Windows ARN.
```

Comment:
I would say that this only applies when deploying windows nodes, otherwise there is no need use this file and I would delete it